pkg/repo/emojis/pg.go

```diff
@@ -0,0 +1,27 @@
+package emojis
+
+import (
+	"gorm.io/gorm"
+
+	"github.com/defipod/mochi/pkg/model"
+)
+
+type pg struct {
+	db *gorm.DB
+}
+
+func NewPG(db *gorm.DB) Store {
+	return &pg{db: db}
+}
+
+func (pg *pg) Create(emojis *model.Emojis) error {
+	return pg.db.Create(emojis).Error
+}
+
+func (pg *pg) GetListEmojis(listCode []string) (model []*model.Emojis, err error) {
+	return model, pg.db.Raw("SELECT id, code, discord_id, telegram_id, twitter_id FROM emojis WHERE code in (?)", listCode).Find(&model).Error
```

Comment:
please check can we use following format, or sth like that
```suggestion
	return model, pg.db.Where("code in (?)", listCode).Find(&model)
```